src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,117 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTORY = 0.75;
+    private int capacity;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    private static class Node<K, V> {
+        private final int hash;
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
```

Comment:
1. `hash` is calculable field
2. you never pass `next` value through constructor
```suggestion
        private final K key;
        private V value;
        private Node<K, V> next;

        public Node(K key, V value) {
```